common/alert/alerts-en.html

```diff
@@ -0,0 +1,144 @@
+---
+{
+	"title": "GCWeb Alert example",
+	"language": "en",
+	"altLangPage": "alerts-fr.html",
+	"breadcrumbs": [
+		{ "title": "GCWeb home", "link": "https://wet-boew.github.io/GCWeb/index-en.html" }
+	],
+	"dateModified": "2023-02-28"
+}
+---
+<div class="wb-prettify all-pre"></div>
+
+<h2>Alert with headings</h2>
+<section class="alert alert-danger">
+	<h3>Danger alert</h3>
+	<p>Alert details.</p>
+</section>
+<section class="alert alert-warning">
+	<h3>Warning alert</h3>
+	<p>Alert details.</p>
+</section>
+<section class="alert alert-success">
+	<h3>Success alert</h3>
+	<p>Alert details.</p>
+</section>
+<section class="alert alert-info">
+	<h3>Info alert</h3>
+	<p>Alert details.</p>
+</section>
+<h3>Code</h3>
+<pre><code>&lt;section class="alert alert-danger"&gt;
+	&lt;h3&gt;Danger alert&lt;/h3&gt;
+	&lt;p&gt;Alert details.&lt;/p&gt;
+&lt;/section&gt;
+
+&lt;section class="alert alert-warning"&gt;
+	&lt;h3&gt;Warning alert&lt;/h3&gt;
+	&lt;p&gt;Alert details.&lt;/p&gt;
+&lt;/section&gt;
+
+&lt;section class="alert alert-success"&gt;
+	&lt;h3&gt;Success alert&lt;/h3&gt;
+	&lt;p&gt;Alert details.&lt;/p&gt;
+&lt;/section&gt;
+
+&lt;section class="alert alert-info"&gt;
+	&lt;h3&gt;Info alert&lt;/h3&gt;
+	&lt;p&gt;Alert details.&lt;/p&gt;
+&lt;/section&gt;</code></pre>
+
+<h2>links inside Alerts</h2>
+<section class="alert alert-success">
+	<h3>(Success Title)</h3>
+	<p>(Success content goes here <a href="#" class="alert-link">link text</a>.)</p>
+</section>
+<section class="alert alert-info">
+	<h3>(Info Title)</h3>
+	<p>(Info content goes here <a href="#" class="alert-link">link text</a>.)</p>
+</section>
+<section class="alert alert-warning">
+	<h3>(Warning Title)</h3>
+	<p>(Warning content goes here <a href="#" class="alert-link">link text</a>.)</p>
+</section>
+<section class="alert alert-danger">
+	<h3>(Danger Title)</h3>
+	<p>(Danger content goes here <a href="#" class="alert-link">link text</a>.)</p>
+</section>
+<h3>Code</h3>
+<pre><code>&lt;section <strong>class=&quot;alert alert-success&quot;</strong>&gt;
+&lt;p&gt; Success content goes here &lt;a href=&quot;#&quot; <strong>class=&quot;alert-link&quot;</strong>&gt;link text&lt;/a&gt;.&lt;/p&gt;
```

Comment:
The heading is missing here:

```suggestion
<pre><code>&lt;section <strong>class=&quot;alert alert-success&quot;</strong>&gt;
&lt;h3>(Success Title)&lt;/h3>
&lt;p&gt; Success content goes here &lt;a href=&quot;#&quot; <strong>class=&quot;alert-link&quot;</strong>&gt;link text&lt;/a&gt;.&lt;/p&gt;
```

Reply:
The requested change was applied